### DISCUSSION TITLE
Rendering a partial tree

### DISCUSSION THREAD

**Discussion by phylome** [jonasthewolf/gsn2x#284] [Ideas]
Firstly, this is a great tool! It perfectly fits my preferred way of diagramming through a textual representation. So far it seems to be the only working solution for GSN of this kind. I hope you'll continue developing this nice software! Seems it might become a good reason for me to start learning Rust.

Since a GSN can become quite large, I wonder if there is already a way to render a partial GSN tree, with the triangular markers that indicate the continuation. I could imagine to provide commandline parameters defining the root node and the depth of the subtree to render.

Cheers,
Florian

**Comment by jonasthewolf** [comment]
Hi @phylome ,

Thanks for the kind feedback.

The idea, currently, is to use the modular extension to keep the size of diagrams manageable.

Please see the documentation for how to use it.

I thought about your idea, but found it too complex to implement yet. Pagination is tricky in my opinion.

Using modular extension also allows generating nice, navigable overview diagrams.

I am curious on your thoughts. 


Best regards,

Jonas
